python/ct/test/time_utils.py

```diff
@@ -0,0 +1,27 @@
+"""Utilities for testing time-sensitive code."""
+
+import contextlib
+import os
+import time
+
+
+def get_timezone_environ():
+    return os.environ.get("TZ", '')
+
+def set_timezone_environ(new_timezone):
+    os.environ["TZ"] = new_timezone
+    time.tzset()
+
+@contextlib.contextmanager
```

Comment:
As per @phad's comment, I'll be making the tests try at least two different timezones, so putting this directly into setUp/tearDown won't work. It seems worthwhile keeping this code in a utility module as this lowers the barrier to re-use and thereby encourages others to use it in their tests, should they also write some code that involves timestamps.